test/e2e/create_cluster_test.go

```diff
@@ -1169,6 +1168,12 @@ func TestCreateClusterV2(t *testing.T) {
 			obj.Annotations[hyperv1.ControlPlaneOperatorV2Annotation] = "true"
 		}
 	}
+	if !e2eutil.IsLessThan(e2eutil.Version418) {
```

Comment:
Using `AtLeast` would be cleaner and it needs to be `e2eutil.AtLeast(e2eutil.Version419)` since this was not backported, which explains why all the `*-4.18` tests failed.

Reply:
Changed the EnsureCustomLabels `atLeast` to `version419`, but it can't be used here inside TestCreateCluster because it will skip the test.

Reply:
nvm, I also added atLeast(419) for `TestCreateClusterV2`